ospo-book/content/en/05-chapter.md

```diff
@@ -1,7 +1,100 @@
+
+* What percentage of employee contributions are merged?
+* What percentage of employee issues are closed without conversation?
+* How many of our employees have maintainer or leadership roles in key open source projects?
+* Have upstream contributions helped modernize tech skills for employees?
+* Which projects do our employees make over 50% of the contributions?
+
+#### Goal 3: Ecosystem Impact
+
+Working with open source is never easy as rival corporations may dominate upstream projects that your organization is interested in, upstream projects may unexpectedly change licenses, and contributor agreements, whether individual or organizational, can be complex to understand and adhere to. Clearly, such challenges can be overcome and often include strategic engagement with the projects your organization aims to benefit from. Open source ecosystems are economic and social systems comprising different companies, motivations, and requirements intended to support production and demands. In an effort to ensure the efficiency and durability of any open source ecosystem, companies must not only monitor the ecosystem's long-term viability but also engage within the ecosystem when problems are identified and stabilization is required. Key questions to consider regarding ecosystem impact include: 
+
+* What percentage of our suppliers provide open source software bills of material?
+* What is the long-term viability of the open source projects we rely on?
+* What is the risk to the ecosystem if an open source project becomes unviable?
+
+#### Goal 4: Organizational Impact
+
+Engagement with open source communities includes working in the upstream to effectively use open source software in organizational products. In this, there is a need to monitor the intake of open source software for infosec, legal, and engineering reasons. Companies can establish software intake processes, working with teams to either technically track or socially consider issues related to open source intake. organization impact can also include working downstream with projects and companies that rely on your organizational products. This can include working to gain a clearer picture of  the open source that is in your shipped products. Organizations can work in securing and regulating their own internal open source processes in an effort to improve product development activities. Key questions to consider regarding organization impact include: 
```

Comment:
```suggestion
Engagement with open source communities includes working in the upstream to effectively use open source software in organizational products. In this, there is a need to monitor the intake of open source software for infosec, legal, and engineering reasons. Companies can establish software intake processes, working with teams to either technically track or socially consider issues related to open source intake. Organization impact can also include working downstream with projects and companies that rely on your organizational products. This can include working to gain a clearer picture of  the open source that is in your shipped products. Organizations can work in securing and regulating their own internal open source processes in an effort to improve product development activities. Key questions to consider regarding organization impact include: 
```